src/find_controls.cpp

```diff
@@ -39,9 +39,9 @@
 #include <cstdlib>
 #include <cstring>
 
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_FIND_ENFORCE)
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_FIND_ONLY_SOLVER)
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_FIND_MODE)
+MIOPEN_DECLARE_ENV_VAR_STR(MIOPEN_FIND_ENFORCE)
```

Comment:
[Notice] modification of this var won't work as expected because it uses custom parsing function, and the library uses "cached" output of that function.

Reply:
removed static from intermediate vars.